src/models/multiBarHorizontalChart.js

```diff
@@ -24,6 +24,7 @@ nv.models.multiBarHorizontalChart = function() {
         , showXAxis = true
         , showYAxis = true
         , stacked = false
+        , reduceXTicks = true // if false a tick will show for every data point
```

Comment:
For new options, please also document them in the big json object in examples/documentation.html